packages/drivers/odsp-driver-definitions/src/errors.ts

```diff
@@ -22,29 +26,33 @@ export enum OdspErrorType {
 	 */
 	snapshotTooBig = "snapshotTooBig",
 
-	/*
+	/**
 	 * Maximum time limit to fetch reached. Host application specified limit for fetching of snapshot, when
 	 * that limit is reached, request fails. Hosting application is expected to have fall-back behavior for
 	 * such case.
 	 */
 	fetchTimeout = "fetchTimeout",
 
-	/*
+	/**
 	 * SPO admin toggle: fluid service is not enabled.
 	 */
 	fluidNotEnabled = "fluidNotEnabled",
 
 	fetchTokenError = "fetchTokenError",
 
-	// This error will be raised when client is too behind with no way to catch up.
-	// This condition will happen when user was offline for too long, resulting in old ops / blobs being deleted
-	// by storage, and thus removing an ability for client to catch up.
-	// This condition will result in any local changes being lost (i.e. only way to save state is by user
-	// copying it over manually)
+	/**
+	 * This error will be raised when client is too behind with no way to catch up.
+	 * This condition will happen when user was offline for too long, resulting in old ops / blobs being deleted
+	 * by storage, and thus removing an ability for client to catch up.
+	 * This condition will result in any local changes being lost (i.e. only way to save state is by user
+	 * copying it over manually)
+	 */
 	cannotCatchUp = "cannotCatchUp",
 
-	// SPO can occasionally return 403 for r/w operations on document when there is a fail over to another data center.
-	// So to preserve integrity of the data, the data becomes readonly.
+	/**
+	 * SPO can occasionally return 403 for r/w operations on document when there is a fail over to another data center.
```

Comment:
Nit: What is SPO? Can we define it here and/or link to some definition?

Reply:
I think I'm going to defer improvements around the descriptions themselves to a follow up PR. Let me know if you feel strongly that we should address it in this PR though!

Reply:
Sounds good. My remaining comments are all very nitpicky - I think the PR looks great!